runelite-client/pom.xml

```diff
@@ -220,6 +220,12 @@
 			<version>4.12</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
This is already a dependency of junit so I wouldn't expect you to have to pull it in?

Reply:
JUnit only has a dependency on `hamcrest-core` which does not contain any `Matcher` implementations.

Reply:
https://junit.org/junit4/javadoc/4.12/org/junit/rules/ErrorCollector.html based on this it should be already in JUnit